Framework/Beamline/src/ComponentInfo.cpp

```diff
@@ -139,6 +152,20 @@ ComponentInfo::componentsInSubtree(const size_t componentIndex) const {
   return indices;
 }
 
+const std::vector<size_t> &
+ComponentInfo::children(const size_t componentIndex) const {
+  static std::vector<size_t> emptyVec;
+
+  if (!isDetector(componentIndex)) {
+    auto numDets = m_assemblySortedDetectorIndices->size();
+    auto index = componentIndex - numDets;
```

Comment:
I thought there is a helper method for getting the index? Can't remember the name.

Reply:
No this `index` is specific to accessing `m_children`